### PR TITLE
Allow for custom elm-format binary names

### PR DESCRIPTION
**Pull request by mariusbutuc**
#### Problem 
Sometimes, the `elm-format` binary can have a different name. As an example, installing `elm-format` [via homebrew](https://github.com/Homebrew/homebrew-core/blob/master/Formula/elm-format.rb) exposes both `elm-format-0.17` and `elm-format-0.18`, but no `elm-format`.

#### Solution
Provide the `elm_format_binary` setting in the Elm Language Support User settings that allows for the custom `elm-format` binary name to be declared. An example might be 

```json
    "elm_format_binary": "elm-format-0.18",
```

#### P.S.
[Naming things is hard](https://martinfowler.com/bliki/TwoHardThings.html)… Not married to the `elm_format_binary` name for such setting. If one can think of one that better conveys intent, I'd be happy to swap them. :v:

### CODE REVIEW

File: Settings/Elm Language Support.sublime-settings [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "debug": false,
 	"enabled": true,
 	"elm_docs_path": "docs.json",
+    "elm_format_binary": "elm-format",
```
Comment (sentience) [inline Settings/Elm Language Support.sublime-settings]: This file has some inconsistent indenting, and your change adds to it. Would you mind please re-indenting this whole file with four spaces?

Comment (mariusbutuc) [inline Settings/Elm Language Support.sublime-settings]: The initial change was minimal not to collide with #14, but it now makes sense to change it here. Good idea! Added in https://github.com/elm-community/Elm.tmLanguage/commit/019afe2464d31c41510587a50734a8f9aba06d37.

**Comment by mariusbutuc** [comment]
@sentience are the current changes good for a merge and release?

**Comment by sentience** [comment]
Thank you @mariusbutuc! Just waiting for #17 to land and then I’ll release a new version of the package.

**Comment by mariusbutuc** [comment]
PS: An alternative (simpler?) to this is to create a symlink:

```shell
$ ln -s /usr/local/bin/elm-format-0.18 /usr/local/bin/elm-format
```

**Pull request merged**